README.md

```diff
@@ -261,3 +261,8 @@ The [examples](examples/naive) directory contains numerous exemplary assembly sn
 
 The use of SLOTHY from the command line is illustrated in [scripts/](scripts/) supporting the real-world optimizations
 for the NTT, FFT and X25519 discussed in [Fast and Clean: Auditable high-performance assembly via constraint solving](https://eprint.iacr.org/2022/1303).
+
+## Usage Tracking
+SLOTHY was used to generate optimized assembly and improve upon previous performance in multiple instances:
```

Comment:
That's true for more examples -- here you want to track 'real world' uses of the SLOTHY generated code, or?